pyproject.toml

```
@@ -22,3 +29,7 @@ exclude = [
     ".tox",
     ".eggs",
 ]
+
+[tool.isort]
+py_version=39
```

Do you know why github flag this section as incorrect?
(should we write the version in string?)
```suggestion
py_version = 39
```

---

Good idea. What did it flag? I didn't see any issues.